### PR TITLE
guided-tour / traduction

### PR DESCRIPTION
**Pull request by pvilalta**
hello team,

This PR brings traductions to the guided-tour

Paul

### CODE REVIEW

File: src/BIMDataComponents/BIMDataGuidedTour/BIMDataGuidedTour.vue [hunk 1]
```diff
@@ -95,7 +95,7 @@
                 color="granite"
                 @click="close"
               >
-                Commencer
+                {{ this.translate("continue") }}
```
Comment (Kurtil) [inline src/BIMDataComponents/BIMDataGuidedTour/BIMDataGuidedTour.vue:98]: this ??

File: src/BIMDataComponents/BIMDataGuidedTour/BIMDataGuidedTour.vue [hunk 2]
```diff
@@ -76,7 +76,7 @@
                 color="granite"
                 @click="close"
               >
-                Skip
+                {{ this.translate("skip") }}
```
Comment (Kurtil) [inline src/BIMDataComponents/BIMDataGuidedTour/BIMDataGuidedTour.vue:79]: this ??

**Review by Kurtil — CHANGES_REQUESTED** [review]

**Comment by Kurtil** [comment]
`this` in template should be remove imo

**Review by NicolasRichel — APPROVED** [review]

**Review by Kurtil — APPROVED** [review]

**Pull request merged**